src/progressive-profiling/OrganizationFormField.jsx

```diff
@@ -56,9 +56,9 @@ const OrganizationFormField = (props) => {
             value: org.id,
             nameOrg: org.label,
             label: (
-              <span>
+              <span style={{ fontSize: '14px' }}>
                 {org.label}
-                <span style={{ fontSize: '0.8em', color: '#888', marginLeft: '8px' }}>
+                <span style={{ color: '#888', marginLeft: '8px' }}>
```

Comment:
[question] Can we use classes instead of inline styles?

Reply:
done